### PR TITLE
Added Russian SCP: SL Redirect, Spanish Clash of Clans Wiki.gg, and new Mario Redirects

### PR DESCRIPTION
**Pull request by DrWhoFan13**
Self-explanatory, I redirected the Russian SCP: SL Fandom to the offical wiki and the Spanish Clash of Clans Fandom to the Spanish Clash of Clans wiki.gg. And now there's several new Mario wiki redirects.

### CODE REVIEW

File: pages/popup/index.html [hunk 1]
```diff
@@ -108,7 +108,7 @@ <h1>Indie Wiki Buddy</h1>
             <input id="crossLanguageCheckbox" type="checkbox" />
             <span id="crossLanguageIcon" aria-hidden="true"></span>
             Redirect non-English Fandom wikis to
-            indie English wikis there is no same-lang wiki
+            indie English wikis when there is no same-lang wiki
```
Comment (KevinPayravi) [inline pages/popup/index.html:111]: You can actually undo this change - we're going to merge in the internationalization PR soon, which will conflict with this change (and will include the fix).

**Comment by DrWhoFan13** [comment]
Removed the typo fix, and glad to hear you're merging that PR

**Review by KevinPayravi — APPROVED** [review]
Thank you!
